float64[] linear_vel
float64 yaw_vel
bool event